src/Console/ConfigurationResolver.php

```diff
@@ -588,6 +588,17 @@ private function parseRules()
             return $this->getConfig()->getRules();
         }
 
+        $rules = trim($this->options['rules']);
+
+        if ($rules[0] === '{') {
+            $rules = json_decode($rules, true);
+            if ($error = json_last_error()) {
+                throw new InvalidConfigurationException("Invalid JSON rules input: Error $error");
```

Comment:
drop checking method existence please
add `symfony/polyfill-php55` to requirements